init_cmd.go

```diff
@@ -18,12 +19,27 @@ There must be a local git repository in order for initialization
 to succeed.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		version := cmd.Parent().Version
-		givenVersion, _ := cmd.Flags().GetString("version")
+		givenVersion, err := cmd.Flags().GetString("version")
+		if err != nil {
+			log.Fatal(err)
+		}
 		if givenVersion != version {
 			version = givenVersion
 		}
 
-		err := client.InitializeInertiaProject(cmd.Parent().Version)
+		// Determine best build type for project
+		buildType := "herokuish"
```

Comment:
I'm not sure whether it's necessary at this time, but I wonder whether it's a good idea to turn our build types into the go-equivalent of an enum?

Reply:
Was looking at the same article as well 😝

If we go with enums, would we put it in a global constants file; lets say... under `common`?

Reply:
Yeah, common would be the right place, since the setting is determined clientside 👍